### PR TITLE
[CIRC-1966]-Api for supporting circulation change.

### PR DESCRIPTION
**Pull request by SinghAdes**
## Purpose
_Describe the purpose of the pull request. Include background information if necessary._
[CIRC-1966](https://issues.folio.org/browse/CIRC-1966)
## Approach
_How does this change fulfill the purpose?_

#### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

## Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

File: src/main/resources/swagger.api/circulation-item.yaml [hunk 1]
```diff
@@ -165,6 +180,34 @@ components:
...
+      required: false
+      description: Skip over a number of elements by specifying an offset value for the query
+    limit:
+      name: limit
+      in: query
+      schema:
+        type: integer
+        default: 1000
+        minimum: 1
+        maximum: 2147483647
```
Comment (gurleenkaurbp) [inline src/main/resources/swagger.api/circulation-item.yaml:208]: Is the minimum and maximum value mentioned here are the standard values for FOLIO? I mean how these values get derived from, should we check with Kalibek about it ?

Comment (SinghAdes) [reply src/main/resources/swagger.api/circulation-item.yaml:208]: @gurleenkaurbp :- Yes these are standard values
https://github.com/search?q=org%3Afolio-org%202147483647&type=code#:~:text=2140-,maximum%3A%202147483647,-2141

Comment (gurleenkaurbp) [reply src/main/resources/swagger.api/circulation-item.yaml:208]: Thanks @SinghAdes

**Review by Vignesh-kalyanasundaram — APPROVED** [review]

**Comment by Vignesh-kalyanasundaram** [comment]
Please update the jira number.

**Review by gurleenkaurbp — APPROVED** [review]

**Pull request merged**